plugins/home/src/homePageComponents/WelcomeTitle/WelcomeTitle.tsx

```diff
@@ -23,10 +23,17 @@ import React, { useEffect, useMemo } from 'react';
 import useAsync from 'react-use/lib/useAsync';
 import { getTimeBasedGreeting } from './timeUtil';
 
-export const WelcomeTitle = () => {
+export interface WelcomeTitleLanguageProps {
+  language?: string;
```

Comment:
Would it make more sense to have an array of Languages? If an Org had two or three languages it uses them you could "pin" this to pick only from one of those

Reply:
I agree this @awanlin here, if we're going to give the option(s) why not allow multiple indeed 👍

Reply:
Added multi language option